docs/source/tutorials/outliers.ipynb

```diff
@@ -528,9 +624,15 @@
     "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
     "\n",
     "# Verify the top identified test outliers data are mostly non-animal images\n",
-    "num_animals = len([i for i in range(len(top_outlier_subset)) if top_outlier_subset[i][1] in animal_classes])\n",
-    "if 1 - (num_animals / len(top_outlier_subset)) < 0.81:\n",
-    "    raise Exception(\"Not enough non-animal images amongst top-ranked outliers in test_data.\")"
+    "top_ood_features_subset = torch.utils.data.Subset(test_data, top_ood_features_idxs)\n",
+    "top_ood_predictions_subset = torch.utils.data.Subset(test_data, top_ood_predictions_idxs)\n",
+    "num_animals = len([i for i in range(len(top_ood_features_subset)) if top_ood_features_subset[i][1] in animal_classes])\n",
+    "if 1 - (num_animals / len(top_ood_features_subset)) < 0.81:\n",
+    "    raise Exception(\"Not enough non-animal images amongst top-ranked outliers in test_data.\")\n",
+    "    \n",
+    "num_animals = len([i for i in range(len(top_ood_predictions_subset)) if top_ood_predictions_subset[i][1] in animal_classes])\n",
+    "if 1 - (num_animals / len(top_ood_predictions_subset)) < 0.50:\n",
```

Comment:
If only 40% of the test data is non-animal, maybe threshold like 0.45 is good enough for CI purposes